deeds-tenant-webapp/src/main/webapp/vue-app/wom-wallet-extensions/components/WoMStatusRewardExtension.vue

```diff
@@ -109,6 +109,9 @@ export default {
         return this.hub?.managerClaimableAmount || 0;
       }
     },
+    formattedClaimableAmount() {
+      return this.claimableAmount % 1 === 0 ? this.claimableAmount : this.claimableAmount.toFixed(2);
```

Comment:
Should use `number-format` like formatting in order to add localized thousands and comma separators. But to fix for future commit, not blocker.